src/context-selection/attribute-option-combo-selector-bar-item/use-attribute-params.js

```diff
@@ -15,10 +15,16 @@ export const useAttributeParams = () => {
         const categoryCombo = selectors.getCategoryComboById(data, catComboId)
 
         const selectedOptions = Object.values(attributeOptionComboSelection)
+
+        if (categoryCombo.isDefault) {
+            // we don't need to supply attributeCombo/options when it's default
+            return {
+                validSelection: true,
+            }
```

Comment:
I'd argue that - despite not being necessary - the returned shape should always be the same.
The values for `attributeCombo` and `attributeOptions` are not undefined here, and I think we should change it for the default return value as well as then components won't have to do "null checks" as long as they rely on the `validSelection`

```suggestion
            return {
                attributeCombo: '',
                attributeOptions: [],
                validSelection: true,
            }
```

Reply:
When undefined these will be omitted from the request when using these. I could set these to `undefined` as well, but we shouldn't need to do any null checks with these values - as they should be sent as is.